src/Compiler/CompilerDialects.cpp

```diff
@@ -46,8 +46,8 @@ DialectRegistry registerDialects(ArrayRef<accel::Accelerator::Kind> accels) {
   for (auto *accel : accel::Accelerator::getAccelerators())
     accel->registerDialects(registry);
 
-  if (useOldBufferization)
-    memref::registerAllocationOpInterfaceExternalModels(registry);
+  memref::registerAllocationOpInterfaceExternalModels(registry);
```

Comment:
Can you comment on why we don't need to execute this conditionally anymore?

Reply:
fixed.

Reply:
Did you upload the latest version? It does not looked conditionally executed.

Reply:
These registrations are needed by both new and old deallocation. Thus, they are NOT conditionally executed.

Reply:
Got it, thanks for the explanation